framework/contrib/hit/src/hit/parse.cc

```diff
@@ -823,6 +828,21 @@ Field::vecIntVal()
   return vec;
 }
 
+std::vector<int64_t>
+Field::vecInt64Val()
+{
+  std::vector<int64_t> vec;
+  for (const auto & s : vecStrVal())
+  {
+    std::istringstream iss(s);
+    int64_t conversion;
+    if ((iss >> conversion).fail() || !iss.eof())
+      throw Error("cannot convert field '" + fullpath() + "' value '" + s + "' to integer");
+    vec.push_back(conversion);
+  }
+  return vec;
+}
```

Comment:
I would make a helper template method that you can instantiate with `int64_t` and `int` to remove the code duplication